src/controllers/contractCategories/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Request, Response } from 'express';
+import ContractCategoryService from '../../services/contractCategories';
+import {
+  CreateContractCategoriesRequest,
+  FindContractCategoriesRequest,
+  PatchContractCategoriesRequest
+} from './types';
+import { handlePrismaError } from '../../lib/handle-prisma-error';
+
+export default class ContractCategoryController {
+  static async getAll(_: Request, response: Response) {
+    const contractCategories = await ContractCategoryService.getAll();
+    return response.success({ data: contractCategories });
+  }
+
+  static async find(request: Request, response: Response) {
+    const { id } = request.params as any as FindContractCategoriesRequest;
+
+    const contractCategory = await ContractCategoryService.find(id);
```

Comment:
is this working? shouldn't it be like the following?

```suggestion
    const contractCategory = await ContractCategoryService.find({ id });
```

Reply:
It's working, i changed because it said that id: string is not assignable to parameter of type 'string'

Reply:
Changed to {id}